src/test/java/seedu/address/storage/JsonAdaptedRoomTest.java

```diff
@@ -1,17 +1,34 @@
 package seedu.address.storage;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.TypicalRooms.ROOM_INVALID_PATIENT_PRESENT_OCCUPIED_FALSE;
 import static seedu.address.testutil.TypicalRooms.ROOM_NO_PATIENT_NO_TASK;
+import static seedu.address.testutil.TypicalRooms.ROOM_PATIENT_ALICE_NO_TASK;
 
 import org.junit.jupiter.api.Test;
 
+import seedu.address.commons.exceptions.IllegalValueException;
+
 class JsonAdaptedRoomTest {
```

Comment:
i think you can consider looking at the `JsonAdaptedPersonTest` class for an example of what to test!